### DISCUSSION TITLE
Different Authentication Methods on Zones

### DISCUSSION THREAD

**Discussion by 1995parham** [emqx/emqx#5459] [Q&A]
We have two different zones, internal and external. is there any way for us to have different authentication methods for these zones? for example using `auth_jwt` for internal and `auth_http` for external?

**Comment by k32** [comment]
Hello,

I briefly looked at the code, and it appears that the same authentication chain is used for all zones. But I think having different chains for different zones could be a useful feature. Could you submit a feature request?

**Reply by 1995parham** [reply]
Yes, I will be happy to do it, by feature request you mean creating a PR on [EIP](https://github.com/emqx/eip) repository?

**Reply by zmstone** [reply]
yes pelase

**Reply by 1995parham** [reply]
Thanks for following this up, I have created the [PR](https://github.com/emqx/eip/pull/40)
